docs/fast_data/tutorials/low-code.mdx

```diff
@@ -484,51 +465,101 @@ It is important to also define the JSON Schema for the _order_ field of the Sing
+Now we need to configure all the parameters of the microservice, here is a set of the fields that we need to configure:
+
+- `SINGLE_VIEWS_COLLECTION`: should be the name of the single view which your single view creator is responsible for (we suggest `order-history`, the same name as the Single View we are going to create);
+- `SINGLE_VIEWS_PORTFOLIO_ORIGIN`: a representative name used in debugging and metrics to identify the source of any Single View update request (we suggest to use the name of the System of Record, `delivery`);
+- `TYPE`: identifies the type of projection changes that need to be read. It should be the same as the Single View name you want to update (also here we are going to call it `order-history`).
+
+Also, based on the connection channel where the microservice reads the projection changes, additional environment variables needs to be set:
+
+<Tabs
+  defaultValue="kafka"
+  groupId="svc-connection"
+  values={[
+      { label: 'Kafka', value: "kafka", },
+      { label: 'MongoDB', value: "mongodb", },
+  ]}
+>
+
+<TabItem value="kafka">
+
+* `PROJECTION_CHANGES_SOURCE`: by using the value `kafka`, it instructs the service to read projection changes from a Kafka connection;
+* kafka connection variables such as:
+  * `KAFKA_BROKERS`: a comma-separated list of your Kafka brokers;
+  * `KAFKA_SASL_USERNAME`, `KAFKA_SASL_PASSWORD` and `KAFKA_SASL_MECHANISM` if authentication needs to be performed
+  * `KAFKA_PROJECTION_CHANGES_TOPIC`: name of the topic that will receive projection changes in the shape of [trigger messages](/fast_data/inputs_and_outputs.md#single-view-trigger-message), you can choose `demo.development.delivery.order-history.trigger`;
+  * `KAFKA_GROUP_ID`: name of the consumer group that will be used to subscribe to the topic from the `KAFKA_PROJECTION_CHANGES_TOPIC` variable.
+
+</TabItem>
+<TabItem value="mongodb">
+
+* `PROJECTIONS_CHANGES_COLLECTION`: it is the name of the collection that includes the Projection Changes generated by the Real-Time Updater, and it must have the same value set in that microservice (we previously suggested `fd-pc-delivery`);
+* `SCHEDULING_TIME`: tells how many milliseconds the microservice needs to wait before polling new projection changes from the `PROJECTION_CHANGES_SCHEMA_COLLECTION`. We set it to `5000`, meaning that 5 seconds will pass between each polling.
+
+</TabItem>
+
+</Tabs>
 
 :::info
-We need to define one strategy for each Projection because when the projection is affected by a change, we need to calculate which single views are impacted.
-This is made possible by the strategies. With the "Low Code Strategy", the system will automatically calculate the impacted single views.
+The Single View Creator includes more optional parameters that might be included via Environment Variables. If you need to know more about them, you can [refer to the related documentation section](/fast_data/configuration/single_view_creator/index.md#environment-variables).
```

Comment:
are parameters needed to make SVC work, or are they not needed for the tutorial?

Reply:
they are optional